bulk_grades/views.py

```diff
@@ -91,6 +91,11 @@ def post(self, request, course_id, *args, **kwargs):
             the_file = request.FILES['csv']
             self.processor.process_file(the_file, autocommit=True)
             data = self.processor.status()
+            data['error_messages'] = [
```

Comment:
Could you split this out into a for loop rather than a comprehension? I think it would be clearer